jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -164,6 +165,14 @@ input[type="password"] {
     border-radius: $border-radius;
 }
 
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
```

Comment:
Only modern browser support number functionality so we should be able to style them all but I will browser test to see if there will be arrows.